source/_patterns/01-molecules/components/social-links.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="social-links">
+<div class="social-links" role="contentinfo" aria-label="Social media links for eLife Sciences">
```

Comment:
I wonder if the`contentinfo` is redundant here, as the footer is already the contentinfo in this context: https://w3c.github.io/aria-practices/examples/landmarks/contentinfo.html#tabpanel2 (The `social-links` pattern is only ever in the site footer.)

Reply:
I see what you mean and agree but as usual there are developers out there who think you should still add it in.

I will remove the role and leave the label if you're happy with that?

Reply:
Could use a comment to clarify why it’s not there?